env.js

```diff
@@ -0,0 +1,96 @@
+const path = require('path');
+const fs = require('fs');
+const child_process = require('child_process');
+const rimraf = require('rimraf');
+
+const promisify = fn => new Promise((res, rej) => {
```

Comment:
please use `util.promisify` here (the npm package), instead of writing your own homegrown promisify function.